validator/static/validator/js/scores_consumer.js

```diff
@@ -0,0 +1,123 @@
+const pyworker = await import((on_gae)?'./py-worker.min.js':'./py-worker.js');
+const asyncRun = pyworker.asyncRun;
+
+const validate_scores = await fetch(new URL('../python/bin/validation_scores.py', import.meta.url, null)).then(response => response.text());
+
+let dirHandle;
+let validateFileHandle;
+
+function startLoading(){
```

Comment:
Should we merge the 2 functions `startLoading` and `finishLoading` into 1, with a flag argument instead ?

Reply:
Yes something like toggleLoading(bool)